kpm/lib/kpm/database.rb

```diff
@@ -34,16 +38,21 @@ def set_database_name(database_name = nil)
       end
 
       def set_mysql_command_line
-        @@mysql_command_line = "mysql #{@@database} --user=#{@@username} --password=#{@@password} -e"
+        @@mysql_command_line = "mysql #{@@database} --user=#{@@username} --password=#{@@password} "
       end
 
-      def execute_insert_statement(table_name, query, qty_to_insert, record_id = nil)
+      def execute_insert_statement(table_name, query, qty_to_insert, table_data, record_id = nil)
 
         if not record_id.nil?
           query = "set #{record_id[:variable]}=#{record_id[:value]}; #{query}"
         end
         query = "SET autocommit=0; #{query} COMMIT;"
-        response = `#{@@mysql_command_line} "#{query.gsub('"','\\"')}" 2>&1`
+        
+        File.open(STATEMENT_TMP_FILE,'w') do |s|
+          s.puts query.gsub('"','\\"')
```

Comment:
Single quotes need to be escaped as well: I just had the issue with a `reason` field containing the word `DIDN'T`. Worth a unit test maybe?